src/backend/columnar/columnar_reader.c

```diff
@@ -149,6 +152,10 @@ ColumnarBeginRead(Relation relation, TupleDesc tupleDescriptor,
 		totalRowCount += stripeMetadata->rowCount;
 	}
 
+	int flags = PVC_RECURSE_AGGREGATES |
+				PVC_RECURSE_WINDOWFUNCS | PVC_RECURSE_PLACEHOLDERS;
```

Comment:
Can we also add some tests which would fail if we didn't set some of these flags?

Reply:
On second thought, why would any of these matter? The only cases we care about are when the min/max can disprove it. If you have something like `HAVING count(x) > 3` or some window function, then a min/max on `x` won't disprove that.

I just copied those flags across, but I can't imagine a situation where they help. And even if we miss one, it won't affect correctness. Thoughts? Am I missing a case?